### PR TITLE
HLM - 325 | Create new consumer in notificaion-service to accept raw messages

### PR DESCRIPTION
**Pull request by ajay-beehyv**
@prasanna-egov  @saiprakash-v @varadeth

### CODE REVIEW

File: backend/notification-service/pkg/consumers/notify_consumer.go [hunk 1]
```diff
@@ -77,3 +78,76 @@ func notifyConsumer() {
...
+				var notificationPayload model.NotificationPayload
+				if err = json.Unmarshal([]byte(string(msg.Value)), &notificationPayload); err != nil {
+					log.Errorf("Received message is not in NotificationPayload format %+v", err)
+				}
+				log.Infof("notificationPayload: %v",notificationPayload)
+				request,err := services.ConstructMessage(notificationPayload)
+				if(err==nil){
+					for _, recipient := range strings.Split(*request.Recipient, ",") {
+						contactType, err := services.GetContactType(recipient)
+						if err != nil {
```
Comment (saiprakash-v) [inline backend/notification-service/pkg/consumers/notify_consumer.go:108]: You can do this err!=nil thing at the end of the above line itself.
`contactType, err := services.GetContactType(recipient); err != nil {...}`
Something like this. Please change it in all the places.

File: backend/notification-service/pkg/consumers/notify_consumer.go [hunk 2]
```diff
@@ -77,3 +78,76 @@ func notifyConsumer() {
...
+				
+			} else {
+				// The client will automatically try to recover from all errors.
+				fmt.Printf("Consumer error: %v \n", err)
+			}
+			if msg != nil {
+				_, err = c.CommitMessage(msg)
+			}
+			if err != nil {
+				log.Errorf("Error in committing notify2 message %+v", err)
```
Comment (saiprakash-v) [inline backend/notification-service/pkg/consumers/notify_consumer.go:148]: Please change `notify2`

**Review by saiprakash-v — CHANGES_REQUESTED** [review]

File: backend/notification-service/pkg/consumers/notify_consumer.go [hunk 3]
```diff
@@ -77,3 +78,75 @@ func notifyConsumer() {
...
+						emailId, err := services.GetEmailId(recipient)
+						if err == nil {
+							if err := services.SendEmail(emailId, request.Subject, *request.Message); err == nil {
+								log.Debugf("Email sent successfully")
+
+							} else {
+								log.Errorf("Error in sending Email %+v", err)
+							}
+						} else {
+							log.Errorf("Invalid notification mobile number %+v, %+v", request, err)
```
Comment (varadeth) [inline backend/notification-service/pkg/consumers/notify_consumer.go:135]: should be Invalid notification email address

**Review by varadeth — APPROVED** [review]

File: backend/notification-service/pkg/consumers/notify_consumer.go (hunk 3 above)
Comment (prasanna-egov) [reply backend/notification-service/pkg/consumers/notify_consumer.go]: I don't see this change is it done?

**Pull request merged**